tools/converter/synapse2ballerina/src/main/java/org/wso2/ei/tools/synapse2ballerina/util/Constant.java

```diff
@@ -30,4 +30,7 @@
     public static final String JSON = "json";
     public static final String XML = "xml";
     public static final String NAMED_SEQUENCE_TYPE = "NAMED";
+    public static final String JMS_PREFIX = "jms:";
+
```

Comment:
Please check these extra lines. Let's make sure that the formatting is done properly